lightly/api/api_workflow_download_dataset.py

```diff
@@ -150,8 +148,72 @@ def lambda_(i):
             msg += 'Failed at image: {}'.format(results.index(False))
             warnings.warn(msg)
 
+    def get_all_embedding_data(self) -> List[DatasetEmbeddingData]:
+        """Returns embedding data of all embeddings for this dataset."""
+        return self._embeddings_api.get_embeddings_by_dataset_id(
+            dataset_id=self.dataset_id
+        )
+
+    def get_embedding_data_by_name(self, name: str) -> DatasetEmbeddingData:
+        """Returns embedding data with the given name for this dataset.
+
+        Raises:
+            ValueError:
+                If no embedding with this name exists.
 
+        """
+        for embedding_data in self.get_all_embedding_data():
+            if embedding_data.name == name:
+                return embedding_data
+        raise ValueError(
+            f"There are no embeddings with name '{name}' for dataset with id "
+            f"'{self.dataset_id}'."
+        )
 
+    def _get_latest_default_embedding_data(self) -> Optional[DatasetEmbeddingData]:
```

Comment:
Let's make this take a list of embeddings and remove it from the api workflwo client
```python
def _get_latest_default_embedding_data(embeddings: list[DatasetEmbeddingData]) -> Optional[DatasetEmbeddingData]